pkg/cluster/manager/display.go

```diff
@@ -237,16 +240,29 @@ func (m *Manager) GetClusterTopology(name string, opt operator.Options) ([]InstI
 			status = ins.Status(tlsCfg, masterActive...)
 		}
 
-		// Query the service status
-		if status == "-" {
+		since := "-"
+		if !opt.ShowUptime {
+			since = formatInstanceSince(ins.Uptime(tlsCfg))
+		}
+
+		// Query the service status and uptime
+		if status == "-" || (opt.ShowUptime && since == "-") {
```

Comment:
Sorry, the previous check is reversed, so the check here is necessary, if user specifies `opt.ShowUptime` but `since` is empty(not implemented or server was down), we try to fetch it via `ssh-systemctl-status`)

Reply:
I means that you have checked this in latter codes:

https://github.com/pingcap/tiup/pull/1231/files#diff-89b17f92d341c20020ab4246afa2eed55ab6aae2ce14541cfd40168e39b0c014R253
https://github.com/pingcap/tiup/pull/1231/files#diff-89b17f92d341c20020ab4246afa2eed55ab6aae2ce14541cfd40168e39b0c014R262

Reply:
No, the check is necessary. 

The previous check is `A or B`, 
because they all need to invoke `operator.GetServiceStatus`,  
and later for condition A I'm trying to get the `status`(if not this condition, don't need to rewrite `status`), and for B to get the `since`.